### PR TITLE
Monitoring

### PR DESCRIPTION
**Pull request by securims**

### CODE REVIEW

File: benchmark/inference.py [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+    # s = time.monotonic()
+    # _ = run_inference_process_pool(x_test=x_test, max_workers=1)
+    # print(f"Inference one worker {time.monotonic() - s}")
+
+    s = time.monotonic()
+    max_workers = 8
+    _ = run_inference_process_pool(x_test=x_test, max_workers=max_workers)
+    print(f"Inference {max_workers} workers {time.monotonic() - s}")
+
+
```
Comment (truskovskiyk) [inline benchmark/inference.py:149]: why do we need this here?

File: monitoring/neptun.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+
+model.fit(
+    x_train, 
+    y_train, 
+    epochs=200, 
+    batch_size=1024, 
+    callbacks= [pruning_callback, neptune_cbk], 
+    verbose=1
+)
```
Comment (truskovskiyk) [inline monitoring/neptun.py:71]: should we wrap this into a function?

Comment (securims) [reply monitoring/neptun.py:71]: done

File: benchmark/inference.py (hunk 1 above)
Comment (securims) [reply benchmark/inference.py:149]: This file contains a different autokeras model which works better than the original. Ignore this file

**Pull request merged**